app/models/user.rb

```diff
@@ -33,7 +33,7 @@ def email_address
     when library_id_user?
       email_from_symphony
     when webauth_user?
-      "#{webauth}@stanford.edu"
+      ldap_email || "#{webauth}@stanford.edu"
```

Comment:
Do we want the `OR` case for this?  we already return the address `"#{webauth}@stanford.edu"` in [ldap_email](https://github.com/sul-dlss/sul-requests/pull/580/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR45).  Although this might be here because that is returning `nil` in that case (as I mention in my comment there).